patroni/postgresql/config.py

```diff
@@ -534,7 +535,8 @@ def _write_recovery_params(self, fd: ConfigWriter, recovery_params: CaseInsensit
                     self._passfile_mtime = mtime(self._pgpass)
                 value = self.format_dsn(value)
             else:
-                value = transform_recovery_parameter_value(self._postgresql.major_version, name, value)
+                value = transform_recovery_parameter_value(self._postgresql.major_version, name, value,
+                                                           self._postgresql.available_gucs)
```

Comment:
> It fails on Postgres v11 and older, because recovery.conf wasn't converted to GUC's yet.
Cool, your review comment came just about the same time I posted the above comment 😆 . Thanks for looking too!

> Good that we run behave tests with many different Postgres version :)
Indeed, what a life saver!